release notes/v0.51.0.md

```diff
@@ -0,0 +1,283 @@
+  const bobKeyPair = await crypto.subtle.generateKey(
+    {
+      name: 'ECDH',
+      namedCurve: 'P-256',
+    },
+    true,
+    ['deriveKey', 'deriveBits']
+  );
+
+  // Derive shared secret for Alice
+  const aliceSharedSecret = await deriveSharedSecret(aliceKeyPair.privateKey, bobKeyPair.publicKey);
+
+  // Derive shared secret for Bob
+  const bobSharedSecret = await deriveSharedSecret(bobKeyPair.privateKey, aliceKeyPair.publicKey);
+
+  // alice shared secret and bob shared secret should be the same
+  console.log('alice shared secret: ' + printArrayBuffer(aliceSharedSecret));
+  console.log('bob shared secret: ' + printArrayBuffer(bobSharedSecret));
+}
+
+async function deriveSharedSecret(privateKey, publicKey) {
+  return crypto.subtle.deriveBits(
+    {
+      name: 'ECDH',
+      public: publicKey,
+    },
+    privateKey,
+    256
+  );
+}
+
+const printArrayBuffer = (buffer) => {
+  const view = new Uint8Array(buffer);
+  return Array.from(view);
+};
+```
+
+</details>
+
+The `sign` and `verify` operations got support for ECDSA algorithm. The `sign` operation allows you to sign a message with a private key, while the `verify` operation allows you to verify the signature with a public key.
+
+Other notable updates and fixes:
+
+- [xk6-webcrypto#68](https://github.com/grafana/xk6-webcrypto/pull/68) fixes a degradation for the sign/verify operations for HMAC algorithm.
+- [xk6-webcrypto#75](https://github.com/grafana/xk6-webcrypto/pull/75), [xk6-webcrypto#76](https://github.com/grafana/xk6-webcrypto/pull/76) refactor webcrypto module to be tread-safe.
+- [xk6-webcrypto#74](https://github.com/grafana/xk6-webcrypto/pull/74) adds JWK import/export support for ECDH and ECDSA. Refactors JWK import/export to use only go standard library.
+
+See [webcrypto's module documentation](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/webcrypto/) for more details.
+
+### Timers globally available [#3589](https://github.com/grafana/k6/pull/3589)
+
+`setTimeout` and co. have been part of the JavaScript ecosystem, probably for as long as it has existed.
+
+In the previous releases we stabilized and made them available through `k6/timers` module. While the module isn't going anywhere and might get more identifiers, `setTimeout` is usually used without importing it. For this reason it is now globally available along `clearTimeot`, `setInterval` and `clearInterval`.
+
+No code needs to be changed, but you no longer need to import `k6/timers` to use this functionality.
+
+## UX improvements and enhancements
+
+- [#3670](https://github.com/grafana/k6/issues/3670) Add the ability to [enable profiling](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#profiling-enabled) via environment variable.
+- [#3682](https://github.com/grafana/k6/pull/3589) An event is emitted when the summary is generated. This new added event is part of a not very stable API, if you intend to use it be aware that we may change it in the future.
```

Comment:
I've also updated the suggestion since it was an external contribution.

@codebien, here is a quick search that you could use https://github.com/search?q=user%3Agrafana+repo%3Ak6+milestone%3A%22v0.51.0%22+-author%3Amstoykov+-author%3Aoleiade+-author%3Ana--+-author%3Acodebien+-author%3Aolegbespalov+-author%3Aandrewslotin+-author%3Ajoanlopez+-author%3Aankur22+-author%3Ainancgumus+-author%3Aszkiba+-author%3Adependabot%5Bbot%5D&type=pullrequests

for identifying all external contributions (at least for the k6 repo). I belive we could incorporate checking such search results (with changing milestone) into the checklist

Reply:
Added as an improvement for the template https://github.com/grafana/k6/pull/3727